src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzer.cs

```diff
@@ -490,10 +490,20 @@ private static void ReportDiagnostics(IOperation operation, SmallDictionary<stri
                 IInvocationOperation iOperation => iOperation.TargetMethod,
                 IObjectCreationOperation cOperation => cOperation.Constructor,
                 IFieldReferenceOperation fOperation => IsWithinConditionalOperation(fOperation) ? null : fOperation.Field,
+                IPropertyReferenceOperation pOperation => GetPropertyAccessor(pOperation.Property, operation),
```

Comment:
Will you be handling `IEventReferenceOperation` in a separate PR? It should be similar change to this one I believe..

Reply:
Yeah, I see it is similar, my plan is: raise PR for property first as it is more important and if time allows add Event accessors into it while it is under review

Reply:
Great, thanks.